lib/client/db/mysql/optionfile.go

```diff
@@ -88,9 +96,10 @@ func (o *OptionFile) Upsert(profile profile.ConnectProfile) error {
 	} else {
 		section.NewKey("ssl-mode", MySQLSSLModeVerifyIdentity)
 	}
-	section.NewKey("ssl-ca", profile.CACertPath)
-	section.NewKey("ssl-cert", profile.CertPath)
-	section.NewKey("ssl-key", profile.KeyPath)
+	// On Windows paths will contain \, which must be escaped to \\ as per https://dev.mysql.com/doc/refman/8.0/en/option-files.html
+	section.NewKey("ssl-ca", strings.ReplaceAll(profile.CACertPath, `\`, `\\`))
```

Comment:
Does this change should be only applied for Windows OS ?

Reply:
No, because the parsing is identical for all platforms and we should escape `\` for all platforms. It is true that outside of Windows `\` are less likely to appear, but in principle, they can.